### DISCUSSION TITLE
2023/01/05/build-cdn/

### DISCUSSION THREAD

**Discussion by giscus** [argvchs/giscus-comments#16] [Announcements]
# 2023/01/05/build-cdn/

https://argvchs.github.io/2023/01/05/build-cdn/

**Comment by DrPhilip425** [comment]
大佬导入CDN仓库的时候basic build setting应该怎么配置呀🤔

**Comment by DrPhilip425** [comment]
大佬我在Netlify导入了CDN仓库后，在*build settings*那我*Build command*填的`npm run build`，在*Publish directory*我填的`public`，其他都没填，但部署一直都是失败了，该怎么解决呀o(TヘTo)

**Reply by argvchs** [reply]
。。。这些都不用填啊

**Reply by DrPhilip425** [reply]
哦哦，成功了，谢谢大佬，是我才疏学浅了😭
